frontend/src/app/main/vote/demoday/step2/page.tsx

```diff
@@ -0,0 +1,150 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import { useRouter } from "next/navigation";
+
+import { AlertSubmit } from "@/app/main/_components/alert-submit";
+import { getTeamList, postTeamVote } from "@/lib/api/vote-demoday";
+import axios from "axios";
+
+import { toast } from "@/hooks/use-toast";
+import { ToastAction } from "@radix-ui/react-toast";
+
+import { Heart } from "lucide-react";
+import "../../../_components/spinner.css";
+
+
+export default function Step2() {
+    const [selectedTeam, setSelectedTeam] = useState<string>("");
+    const [teamList, setTeamList] = useState<string[]>([]);
+    const [isLoading, setIsLoading] = useState<boolean>(false);
+    const router = useRouter();
+
+    // 팀 리스트 가져오기
+    const getTeam = async () => {
+        try {
+            const res = await getTeamList();
+            // 팀 리스트가 배열 형태로 오는지 확인
+            if (Array.isArray(res.data)) {
+                setTeamList(res.data);
+            } else {
+                throw new Error("Invalid data format");
+            }
+        }
+        catch (err) {
+        }
+    }
+
+    useEffect(() => {
+        getTeam();
+    }, []);
+
+    // 투표할 팀 정하기
+    const handleVote = (teamName: string) => {
+        if (teamName === selectedTeam) {
+            setSelectedTeam("");
+        } else {
+            setSelectedTeam(teamName);
+        }
+    };
```

Comment:
정말 사소하긴 하지만 이렇게 간략히 쓰는거슨.. 어떨까요
```suggestion
const handleVote = (teamName: string) => {
    setSelectedTeam(prev => (prev === teamName ? "" : teamName));
};
```